libs/datasets/sources/covid_tracking_source.py

```diff
@@ -130,13 +130,25 @@ def standardize_data(cls, data: pd.DataFrame) -> pd.DataFrame:
         # TODO(chris): Handle this in a more sane way.
         data = data.loc[data.state != "PR", :]
 
+        # Removing bad data from Delaware.
+        # Once that is resolved we can remove this while keeping the assert below.
+        icu_mask = data[cls.Fields.IN_ICU_CURRENTLY] > data[cls.Fields.CURRENT_HOSPITALIZED]
```

Comment:
Can you add a sentry error here that reports how many states/rows this affects?
could look something like this: https://github.com/covid-projections/covid-data-model/blob/master/libs/build_processed_dataset.py#L91-L95. 

Would be good to know which states this is happening to

Reply:
How detailed do we want to know? The example given is a pretty simple string do we want to dump all lines of the data frame where we drop a specific value?

Reply:
Or would a simple "X number of lines were changed due to icu correction"

Reply:
Good question! Maybe something like `X number of rows in N states...`?

Reply:
also, I'm fine to punt on this for now, getting this fix out definitely the priority!